packages/mattermost-redux/src/types/client4.ts

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-// I assume these are the log levels
-export type LogLevel = 'ERROR' | 'WARNING' | 'INFO';
+export enum LogLevel {
```

Comment:
I love seeing more enums. We need to do some more of this at some point to get rid of a lot of constants